### DISCUSSION TITLE
Does this official effort from .NET Foundation make Vanara obsolete?

### DISCUSSION THREAD

**Discussion by nsgnkhibdk2cls0f** [dahall/Vanara#367] [Q&A] [answered]
https://github.com/dotnet/pinvoke

Update: Microsoft has a newer effort to make everything even more easy and straight forward: https://github.com/microsoft/win32metadata

**Comment by dahall** [comment, marked as answer]
I've used both of the above and have continued Vanara development due to the inherent weaknesses of those projects to create consistent and usable enumerations, structures and method parameters. Parsing the .h files, even with guidance, is work that will take many years due to the countless inconsistencies in those header files.
